cypress/support/step_definitions/dictionary/createVersion/createVersion.ts

```diff
@@ -0,0 +1,57 @@
+import { Given, Then, When } from "cypress-cucumber-preprocessor/steps";
+import {getDictionaryId, getUser, getVersionId} from "../../../utils";
+
+Given("the user is on the dictionary page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/`);
+  cy.findByText("Versions").should("be.visible");
+});
+When("the user clicks the create new version button", () => {
+  cy.findByRole("button", { name: /Create new version/i }).click();
+});
+Then("the user should be on the create new version dialog box", () =>
+  cy.get("h2").contains("Create new version")
+);
+
+Given("the user is on the create new version dialog box", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/`);
+  cy.findByRole("button", { name: /Create new version/i }).click();
+  cy.get("h2").contains("Create new version");
+});
+When("the user enters the version information", () => {
+  cy.findByLabelText("ID").type("1");
+  cy.get("#released").type("{enter}");
+});
+When("the user submits the form", () => {
+  cy.get("#versionForm").submit();
+});
+Then("the new version should be created", () => {
+  cy.get("table").should("exist");
+  cy.get("table tbody tr").should("have.length", 1);
+  cy.get("table").contains("td", "1");
+  cy.get("table").contains("td", "No");
+});
```

Comment:
There are probably too many assertions here we don't need. E.g., what we care about is there is a row with the ID we filled in. Also, I'm not sure what the "No" here is supposed to match, but we'd probably need to check the actual checkbox if its the release status.